Python/Github_Issues_PRs_Scrapper/requirements.txt

```diff
@@ -0,0 +1,3 @@
+PyGithub
```

Comment:
Please include the version number here. Use `pip freeze` for that

Reply:
Done!